bin/sync_vsc_email_postfix.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+#
+# Copyright 2013-2019 Ghent University
+#
+# This file is part of vsc-administration,
+# originally created by the HPC team of Ghent University (http://ugent.be/hpc/en),
+# with support of Ghent University (http://ugent.be/hpc),
+# the Flemish Supercomputer Centre (VSC) (https://www.vscentrum.be),
+# the Flemish Research Foundation (FWO) (http://www.fwo.be/en)
+# and the Department of Economy, Science and Innovation (EWI) (http://www.ewi-vlaanderen.be/en).
+#
+# https://github.com/hpcugent/vsc-administration
+#
+# All rights reserved.
+#
+"""
+This script creates the list of canonical email adresses for VSC users.
+"""
+from __future__ import print_function
+
+import logging
+import sys
+
+from vsc.accountpage.client import AccountpageClient
+from vsc.accountpage.sync import Sync
+from vsc.accountpage.wrappers import mkAccount
+
+
+class VscPostfixSync(Sync):
+    CLI_OPTIONS = {
+        postfix_address_book: ('Location of the postfix canonical address file', None, 'store', None),
```

Comment:
`address_book` -> `alias_map`, and you can set a default `/etc/postfix/alias` or something like that